src/RadixDlt.NetworkGateway.GatewayApi/gateway-api-schema.yaml

```diff
@@ -1898,6 +1898,31 @@ paths:
         "4XX":
           $ref: "#/components/responses/ErrorResponse"
 
+  /statistics/page/resource-owners/:
```

Comment:
Possibly `/extensions/resource-owners/page` (in that it's a page under the `resource-owners` extension)?

Reply:
we follow page/resource/ pattern in all other places, so i think we should stick with that.

Reply:
If we're using the `/extensions/` prefix, I'd like each extension to be conceptually separate (i.e. each extension `X` would serve routes under `/extensions/X/**`, which could let us have the extensions live elsewhere), and aligned with V2 design.

I'd rather that over aligning with the Gateway URI structure.